cliquet/views/oauth.py

```diff
@@ -23,9 +22,14 @@
                 path='/fxa-oauth/token',
                 error_handler=errors.json_error_handler)
 
+DEFAULT_STATE_EXPIRY_SECONDS = 3600  # 1 hour
```

Comment:
I've been amazed how long the oauth dance can take if the user e.g. forgets their password or has to sign up for a new account, and then wait for the email to come through, and then fish it out of their spam folder, and then click the link but it opens in a different browser, etc etc....